### PR TITLE
Add post-processing scripts for DB download and geojson conversion

### PR DESCRIPTION
**Pull request by wronk**
Adding:
* download.js to fetch the stored results and save to local disk as a CSV file
* csv_to_geojson.py to convert the CSV results to a geojson

### CODE REVIEW

**Comment by wronk** [comment]
@drewbo, should I add the entry points [here](https://github.com/developmentseed/chip-n-scale-queue-arranger/blob/master/package.json#L14)?

If yes, should the format be any different for the python file? I can also add notes to the README once this is ready to go.

File: scripts/csv_to_geojson.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import sys
+import csv
+import json
+
+from mercantile import feature, Tile
+from geojson import Feature
+from pygeotile.tile import Tile as Pygeo_tile
+
+with open(sys.argv[1], 'r') as csvfile:
+    with open('results.geojson', 'w') as results:
```
Comment (drewbo) [inline scripts/csv_to_geojson.py]: can this file name be an argument (or pipe to process.stdout)?

**Review by drewbo — CHANGES_REQUESTED** [review]

**Comment by drewbo** [comment]
I'd add the download function as a yarn script, the python one is dicier since we can't (easily) guarantee the python env from the package.json

File: scripts/csv_to_geojson.py (hunk 1 above)
Comment (wronk) [reply scripts/csv_to_geojson.py]: Yea, argparse the way to go here? I like click but maybe it's good to avoid an extra dependency?

**Comment by wronk** [comment]
@drewbo, added functions to yarn and updated the README

File: README.md [hunk 2]
```diff
@@ -71,6 +71,20 @@ yarn sqs-push [tiles.txt] [https://your-queue-url]
 
  The first argument, `tiles.txt`, is a line-delimited file containing your tile indices in the format `x-y-z` and the second argument is the URL of your SQS Queue. If you have a lot of tiles to push to the queue, it's best to run this script in the background or on a separate computer.  The maximum number of simultaneous inflight SQS requests can be set with the `PROMISE_THRESHOLD` environment variable.
 
+## Post processing
+
+Once the processing is complete, you can pull down the stored results as a simple CSV file.
+
+```sh
+DATABASE_URL='postgres://myusername:mypassword@your-db-string.rds.amazonaws.com:5432/ResultsDB' yarn download my_csv_filename.csv
```
Comment (drewbo) [inline README.md:79]: Can we construct the DB_URL for the user (should be either saved in env or available from config somehow

**Review by drewbo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (wronk) [reply README.md:79]: It's not unwieldy now, but that would be a little nicer. Then we should also probably update the `yarn setup [DB_STRING]` step too. I'm no good with JS though, so might need help to do that

Comment (drewbo) [reply README.md:79]: ah yes, I guess that requires waiting for the CF output. Good to merge now then

**Review by drewbo — APPROVED** [review]

File: scripts/csv_to_geojson.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            results.write('{"type":"FeatureCollection","features":[')
+            next(reader)  # Skip header
+
+            for row in reader:
+
+                # Load as pygeotile using TMS coords
+                geot = tile_func(*[int(t) for t in row[0].split('-')])
+
+                # Create feature with mercantile
+                feat = feature(Tile(geot.google[0], geot.google[1], geot.zoom))
```
Comment (drewbo) [inline scripts/csv_to_geojson.py:55]: @wronk can you correct this line to use TMS (which is what mercantile expects) rather than google?

Comment (wronk) [reply scripts/csv_to_geojson.py:55]: @drewbo, I think mercantile expects X/Y/Z (google) format

https://mercantile.readthedocs.io/en/latest/api/mercantile.html#mercantile.feature

Comment (drewbo) [reply scripts/csv_to_geojson.py:55]: XYZ isn't google; I just ran it on a large data set and it put all my tiles in the ocean

Comment (drewbo) [reply scripts/csv_to_geojson.py:55]: ...because XYZ is google and I was using the wrong input type. That's my bad, this is good to merge as is

**Pull request merged**